### ISSUE TITLE
CC v17c Wooden repeaters not drawing lumen (?)

### ISSUE THREAD

**Issue by kutschkem** [ReikaKalseki/Reika_Mods_Issues#1551]
What am I doing wrong here? I am trying to use Pylon Surge Protection ritual to make the Ritual altar receive Kuro lumen to eventually blow up the wooden repeaters - but it doesn't draw the lumen? I am a little stuck, what is the intended way to blow the repeaters up?

![image](https://user-images.githubusercontent.com/1106888/29252034-5e05ae60-8060-11e7-9d2a-35d6facdee08.png)

**Comment by ReikaKalseki** [comment]
That does not look like it has LOS, and I know they work because others are using them.

**Comment by kutschkem** [comment]
Let this issue closed, but I wanted to let you know that I circumvented my issue by using a storage crystal charger instead of the ritual table - however, I am still unable to get the ritual table to work, with wooden or crystal repeaters. Pointing my manipulator at the table shows a yellow dot - not sure what this means, but I assume that means the structure is not correct? I have rebuilt it several times though.

Has the progression changed in a way that although I have the ritual structure unlocked in v16, I can't use it now that I am in v17? Behind what progression steps is the ritual altar (structure) locked in v17?
